public/js/utils.js

```diff
@@ -89,3 +89,10 @@ function submitTransaction () {
     }
   })
 }
+
+function validateChainId(chainId) {
+  var currId = metamask.version.network;
+  if (currId !== chainId) {
+    alert("WARNING: Metamask is connected to a different Ethereum network\nPlease configure correct RPC url for the ewasm testnet:\nhttp://ewasm.ethereum.org:8545");
```

Comment:
I think the configuration could have a field for either:
a) `rpcUrlSuggestion` - if it is non-empty, then the "Please configure correct RPC..." text is displayed with it
b) `invalidChainConnected` to contain the entire message displayed here

Reply:
added `rpcUrlSuggestion` to configuration